### PR TITLE
Updating icon field on issue model

### PR DESCRIPTION
**Pull request by annalee**
Changing the icon field on the issue model to an FK to media, to make icons easier to re-use without duplicating.

### CODE REVIEW

File: peacecorps/peacecorps/migrations/0019_auto_20141028_1736.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    dependencies = [
+        ('peacecorps', '0018_issue_icon'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='issue',
+            name='icon',
+            field=models.ForeignKey(help_text='A small icon to represent the issue on the landing page.', to='peacecorps.Media', related_name='iconissues'),
```
Comment (msecret) [inline peacecorps/peacecorps/migrations/0019_auto_20141028_1736.py:17]: Do we have any guidelines on keeping lines to less the 80 chars long?

**Comment by msecret** [comment]
LG to me, although maybe one of the backend developers should be the one that merges it.

File: peacecorps/peacecorps/migrations/0019_auto_20141028_1736.py (hunk 1 above)
Comment (cmc333333) [inline peacecorps/peacecorps/migrations/0019_auto_20141028_1736.py:17]: Generally, PEP8 recommends 80 chars, but this is file is auto-generated

**Comment by cmc333333** [comment]
Something to keep in mind - fking to the generic "Media" model means that it's on the CSS/Markup to size appropriately.

**Comment by cmc333333** [comment]
@msecret - note that this requires you wipe your db to upgrade

**Pull request merged**